### ISSUE TITLE
AEO-Light 2.3.3  

### ISSUE THREAD

**Issue by dan7455** [usc-imi/aeo-light#22]
Hello!
How can I change the resolution of 640 * 480 video to 1440 * 1080 when outputting?
I scan with a resolution of 2592 * 1944, and I receive at the output 640 * 480

**Comment by retokromer** [comment]
I guess, it’s hard coded:

https://github.com/usc-imi/aeo-light/blob/6b674455f676d7994202911fd033a94e7faf028a/videoencoder.cpp#L157-L158

Possibly you could change the values and compile (not tested).

**Comment by dan7455** [comment]
Reto Kromer Thank you!
I'm not a programmer, who can help recompile 1440*1080?

**Comment by wilsbacher** [comment]
The simplest path forward for producing sync'd video is to run your own FFmpeg scripts once the AEO wav is finished. Once you have this method down, you'll be able to generate whatever size variants you'd like without being a programmer. 

FFmpeg.org has instructions for downloading the program.  Once you have it installed, then install FFCommand Engine (by Colorlab on Githbub). Open it and look at the code in the "Preset-->ffmprovisr" for "merge audio and video tracks"  this gives you the basic structure of the merge code in FFmpeg:

ffmpeg -y -i video_file -i audio_file   -map 0:v -map 1:a -c copy output_file

**Comment by dan7455** [comment]
Request to the authors of the project to include support for files.

**Comment by sjohnson-imi** [comment]
> wilsbacher Thank you!
> 
> Request to the authors of the project to include support for JPG files.
> When scanning in TIFF we get very large volumes.

Since this is a new issue/feature request, I've started a new issue for it: https://github.com/usc-imi/aeo-light/issues/24

**Comment by ziggyphenocious** [comment]
Where are the AEO log files stored, or are they?  Sometimes I find a setting that works well, and I would like to be able to retrieve it later after the soundtrack extraction.

**Comment by ziggyphenocious** [comment]
I have used the store settings button, but sometimes I just need the values that show up in the .aeolight.log.txt file.  If I retrieve settings for another similar job, I don't want it to overwrite the frame count of the new project.
